proto/cosmwasm/wasm/v1/types.proto

```diff
@@ -19,12 +19,16 @@ enum AccessType {
   // AccessTypeNobody forbidden
   ACCESS_TYPE_NOBODY = 1
       [ (gogoproto.enumvalue_customname) = "AccessTypeNobody" ];
-  // AccessTypeOnlyAddress restricted to an address
+  // AccessTypeOnlyAddress restricted to a single address
+  // Deprecated: use AccessTypeAnyOfAddresses instead
```

Comment:
Why deprecate?
I think it is fine to use the simpler version for the cases where it works